### ISSUE TITLE
The directory you selected already exists and contains an installation. Choose a different target for installation.

### ISSUE THREAD

**Issue by baconpie** [suratovvlad/qbittorrent-installer#20]
Of course it already exists. I'm updating. Let me update.

**Comment by Thermalzombie** [comment]
Do we just delete the existing directory?  Will settings be saved?

**Comment by suratovvlad** [comment]
> Do we just delete the existing directory? Will settings be saved?

Yes, they should be save, but you could check them before uninstall process in %LOCALAPPDATA% or %APPDATA% in folder named qbittorrent

**Comment by suratovvlad** [comment]
> Of course it already exists. I'm updating. Let me update.

I know it is awkward, but it is designed in such way. 
https://forum.qt.io/topic/56103/update-an-installation-with-an-offline-installer-setup-of-qt-installer-framework

I don't have an ability to host a website at this moment, but there are workarounds, I save them here for future updates

https://stackoverflow.com/questions/21389105/qt-installer-framework-offline-update-how
https://stackoverflow.com/questions/46455360/workaround-for-qt-installer-framework-not-overwriting-existing-installation
